GooglePay/src/main/java/com/braintreepayments/api/GooglePayClient.java

```diff
@@ -98,7 +98,7 @@ public void onResult(@Nullable Configuration configuration, @Nullable Exception
                 }
 
                 if (activity == null) {
-                    callback.onResult(false, new GoogleApiClientException(GoogleApiClientException.ErrorType.NotAttachedToActivity, 1));
+                    callback.onResult(false, new IllegalArgumentException("Activity cannot be null."));
```

Comment:
Should we note that this exception type returned as changed in the CHANGELOG?

Reply:
Added this 👍